packages/react/src/date-range-picker/DateRangePicker.test.tsx

```diff
@@ -260,48 +260,46 @@ describe('DateRangePicker', () => {
 		);
 
 		let inputs = container.querySelectorAll('input');
-		let calendarTriggers = container.querySelectorAll('button');
 
 		// The inputs should be a formatted display value of `initialValue`
 		expect(inputs[0]).toHaveValue(fromDateString);
 		expect(inputs[1]).toHaveValue(toDateString);
 
 		// The calendar button trigger should have an aria-label with the formatted display value of `initialValue`
```

Comment:
Not for this PR but it's a pretty good indication that you need another test when you have comments prefacing your eight assertions 😅